src/styles/card.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  display: flex;
+  flex-direction: column;
+
+  &__image {
+    height: 134px;
+    width: 160px;
+    margin: 32px 19px 0;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-text-color;
+    margin: 40px 16px 0;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $text-color;
+    margin: 4px 0 0 16px;
+  }
+
+  &__reviews {
+    display: flex;
+    flex-direction: row;
+    box-sizing: border-box;
+    justify-content: space-between;
+
+    margin: 16px 16px 0;
+
+    &-number {
+      font-weight: 400;
+      font-size: 10px;
+      line-height: 14px;
+      color: $main-text-color;
+    }
+  }
+
+  &__price {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin: 24px 16px 0;
+
+    &-text {
+      font-weight: 400;
+      font-size: 12px;
+      line-height: 18px;
+      color: $text-color;
+    }
+
+    &-cost {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+      color: $main-text-color;
+    }
+  }
+
+  &__link {
+    background-color: $blue-color;
+    width: 166px;
+    height: 40px;
+    box-sizing: border-box;
+    color: $white-color;
+    border-radius: 5px;
+    border: none;
+    text-transform: uppercase;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 40px;
+    margin: 16px;
```

Comment:
The width of `&__link` should be `100% - 32px` to account for the horizontal padding, ensuring the button stretches to the edges of the card while maintaining the padding. Use `calc(100% - 32px)` for the width.